jobs/config.json

```diff
@@ -8307,6 +8307,28 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-upgrade-1.6-1.7-init": {
```

Comment:
nit: Now that I'm thinking about it, we don't actually need to indicate that this is using the init method as that's the only thing that works.
The same goes for future v1.7->v1.8 jobs, `kubeadm upgrade` is the only variant in any case soo....